tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js

```diff
@@ -196,7 +196,7 @@ module.exports = {
           }
         )
         // to hide the access details dialog from the screen, click on the collaborator avatar
-        await this.api.elementIdClick(collaboratorAvatarElId)
+        await this.api.elementIdClick(collaboratorAvatarElId).pause(500)
```

Comment:
:cry: the `waitForElementNotPresent` is exactly for such waits.
```suggestion
        await this.api.elementIdClick(collaboratorAvatarElId)
```

Reply:
yes, that's why I have to use explicit wait. Time waiting for **waitForElementNotPresent** is not enough so it fails intermittently.

Reply:
if this function `waitForElementNotPresent` did not wait enough you might need to increase the timeout for that function.
looks like the function `waitForElementNotPresent` below has timeout `waitForNegativeConditionTimeout` which is `300 ms` as per the config. https://github.com/owncloud/web/blob/a4bfc0090776ad384148259d96fedbe5858607b4/tests/acceptance/nightwatch.conf.js#L47-L48


You can use `waitForConditionTimeout` instead of `waitForNegativeConditionTimeout` for function `waitForElementNotPresent` below.(No need to use explicit pause)

And check if it affects other tests runtime. CC @saw-jan ?

Reply:
> Regarding increasing waitForNegativeConditionTimeout can we afford increasing time for this variable 

I am afraid we should increase the time for `waitForNegativeConditionTimeout`.

>That's a nice fix but what is the use of naming two variables waitForNegativeConditionTimeout and waitForConditionTimeout if we are going to use it in a zigzag pattern like using waitForConditionTimeout in a negative condition like waitForElementNotPresent?

The purpose of `waitForNegativeConditionTimeout` is not to use in commands like `waitForElementNotPresent` or `waitForElementNotVisible` (You got confused here).

The main purpose is to use `waitForNegativeConditionTimeout` in commands like `isVisible` (to determine the visibility) if we expect that the element should not be visible (**negative**).

Reply:
I found that people are using these two variables according to the time needed. So i think it ok to use

Reply:
For example here: https://github.com/owncloud/web/blob/c73b8d03264e4a0e1456a512831eba008ac9579a/tests/acceptance/pageObjects/personalPage.js#L301-L317